_pages/index.html

```diff
@@ -9,24 +9,19 @@
 <body>
     <!-- Container 1  (5 protein morning snacks) -->
     <div> 
-        <h1>
-            <strong>
-            15 Protein-Packed Meals That Are 
-            <em>Easy
-            </em>to Make!
-        </strong>
+        <h1 class="title">
+            15 Protein-Packed Meals That Are <em>Easy</em> to Make!
         </h1>
-        <img src="/Users/safiya/Desktop/protein meal.jpeg" width="450" height="250">
```

Comment:
This images source location is on your laptop and not in the repository. If someone uses this repo, they will not be able to see the image